src/graphql/data/nft/Asset.ts

```diff
@@ -117,22 +116,46 @@ export function useAssetsQuery(
   last?: number,
   before?: string
 ) {
-  const queryData = useLazyLoadQuery<AssetQuery>(assetQuery, {
-    address,
-    orderBy,
-    asc,
-    filter,
-    first,
-    after,
-    last,
-    before,
+  const queryResult = useQuery(assetQuery, {
```

Comment:
in `types-and-hooks` you should find a custom hook for this query 😄 

```
useAssetQuery({
  variables: { ... },
  pollInterval: 999
})
```

Reply:
also i would use the `pollInterval` config over `startPolling`

Reply:
oh just seeing this https://github.com/Uniswap/interface/pull/5077#issuecomment-1302959247